enzyme/tests/TransactionCreate.test.tsx

```diff
@@ -1,18 +1,61 @@
-
+// tslint:disable no-unused-expression
```

Comment:
Would it be possible to have a look at how we get our tslint rules working with the React stuff - that way we wouldn't need to do inline stuff like this!